src/fsharp/FSharp.DependencyManager.Nuget/FSharp.DependencyManager.ProjectFile.fs

```diff
@@ -40,9 +41,18 @@ module internal ProjectFile =
         |> Array.distinct
 
     let findReferencesFromResolutions (resolutions:Resolution array) =
+
+        let equals (s1:string, s2:string) =
+            let s1 = if String.IsNullOrEmpty(s1) then "" else s1
+            let s2 = if String.IsNullOrEmpty(s2) then "" else s2
```

Comment:
This shouldn't be necessary.

Reply:
You know, what, I never knew that string.Compare was okay with null arguments.  Thanks.